CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 # Which target should be build by default?
 set(DEFAULT_BUILD_TYPE "Release")
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Apart from our general disagreemnent on this topic, I would be needing c++14, to avoid writing constructors for structs that have default values. Since the default values do note make any sense in some cases (because default construction does not make sense), I would need to check if c++14 is still needed.

Reply:
Due to serialization the default constructor is unavoidable for these objects. I would prefer to leave c++14 in there. What speaks against c++14, that is reasonably old standard (gcc 5 supports that fully)